src/HtmlUtils.tsx

```diff
@@ -79,8 +79,8 @@ function mightContainEmoji(str: string): boolean {
  * @return {String} The shortcode (such as :thumbup:)
  */
 export function unicodeToShortcode(char: string): string {
-    const shortcodes = getEmojiFromUnicode(char).shortcodes;
-    return shortcodes.length > 0 ? `:${shortcodes[0]}:` : '';
+    const shortcodes = getEmojiFromUnicode(char)?.shortcodes;
+    return shortcodes && shortcodes.length > 0 ? `:${shortcodes[0]}:` : '';
```

Comment:
```suggestion
    return shortcodes?.length > 0 ? `:${shortcodes[0]}:` : '';
```
Shouldn't this work?

Reply:
I think so, but it's helpful for readability to not have to contemplate whether `undefined > 0` :dizzy:

Reply:
I tend to do just do foo?.length and skip the comparison as 0 is falsey and 1+ is truthy